### ISSUE TITLE
Pre-trained weights download mirror

### ISSUE THREAD

**Issue by kbarni** [YunYang1994/TensorFlow2.0-Examples#32]
Please upload the pre-trained weights on an alternative mirror. 

I don't understand a thing from the baidu site and it doesn't let me download the .h5 files... (it seems that it asks to install a downloader...?)

**Comment by YunYang1994** [comment]
yes, you should install a BaiduCloud first and then register in .

**Comment by kbarni** [comment]
...which is not a really a good practice for open projects IMHO.

Can you please upload those files to another location too?

**Comment by YunYang1994** [comment]
yes, I will
